desc/magnetic_fields/_current_potential.py

```diff
@@ -1472,19 +1479,30 @@ def surfint_Bnsqd(phi_mn, I, G):
         * ne_mag
         * eval_grid.weights
     )
-    if not vacuum:  # get Bn from plasma contribution
-        Bn_plasma = compute_B_plasma(eq, eval_grid, vc_source_grid, normal_only=True)
-        Bn_plasma = Bn_plasma * ne_mag * eval_grid.weights
-
-    else:
-        Bn_plasma = jnp.zeros_like(Bn_GI)  # from plasma current, currently assume is 0
+    # get Bn from plasma current contribution
+    Bn_plasma = (
+        jnp.zeros_like(Bn_GI)
+        if vacuum
```

Comment:
nit: im not crazy about this kind of inline if/else if it still takes up multiple lines